src/dotnet/Chat.Service/Conversations.cs

```diff
@@ -15,4 +16,23 @@ public virtual async Task<Conversation[]> GetTile(Session session, ChatId chatId
 
         return await Backend.GetTile(chatId, idTileRange, cancellationToken).ConfigureAwait(false);
     }
+
+    public virtual Task<Conversation?> Get(ConversationId conversationId, CancellationToken cancellationToken)
+        => Backend.Get(conversationId, cancellationToken);
+
+    public virtual async Task<Conversation?> ReSummarize(ConversationId conversationId, CancellationToken cancellationToken)
```

Comment:
All actions we define as Command object. Please, refactor this. Also add Session property to the command and verify owner permissions.